app/mailers/application_mailer.rb

```diff
@@ -4,8 +4,9 @@ class ApplicationMailer < ActionMailer::Base
 
   def dwp_is_down_notifier
     to = Settings.mail.dwp_notification_alert
-    mail(to: to, subject: t('email.dwp_alert_notification..subject')) do |format|
-      format.text { render plain: t('email.dwp_alert_notification.message') }
-    end
+    mail(to: to, subject: t('email.dwp_alert_notification.subject',
+      environment: ENV['ENV'])) do |format|
```

Comment:
Why have an environment variable called 'ENV' ?  Or am I misreading this ?  Its just not descriptive at all ?